tui/common.go

```diff
@@ -52,6 +52,11 @@ type Page interface {
 }
 
 const (
+	// TermWidth is our desired terminal width
+	TermWidth = 80
```

Comment:
this is not actually the terminal size, but the window size.

Reply:
Renaming to WindowWidth and WindowHeight